Source/BugsnagConfiguration.h

```diff
@@ -61,8 +61,7 @@ typedef void (^BugsnagOnErrorBlock)(BugsnagEvent *_Nonnull report);
  *
  *  @return YES if the report should be sent
  */
-typedef bool (^BugsnagOnSendBlock)(NSDictionary *_Nonnull rawEventData,
-                                       BugsnagEvent *_Nonnull reports);
+typedef bool (^BugsnagOnSendBlock)(NSDictionary *_Nonnull rawEventData, BugsnagEvent *_Nonnull reports);
```

Comment:
I think it'd make sense to remove `rawEventData` and rename `reports` as `event`, as it could be a bit confusing having two parameters. In the past users have edited the `rawEventData` and been surprised that changes are not reflected in reports sent to the bugsnag dashboard.

Reply:
Thanks for the background.  Happy to change it, if we're happy that nothing exclusively in the raw data could influence the return value.